ibm/mas_devops/roles/cp4d_install/templates/cpd40/ibmcpd.yaml

```diff
@@ -7,5 +7,7 @@ spec:
   license:
     accept: true
     license: Standard
+  # both storage classes for cpd should be ibmc-file-gold-gid
+  # https://www.ibm.com/docs/en/cloud-paks/cp-data/4.0?topic=requirements-storage
   storageClass: "{{ cpd_storage_class }}"
-  zenCoreMetadbStorageClass: "{{ cpd_block_storage_class }}"
+  zenCoreMetadbStorageClass: "{{ cpd_storage_class }}"
```

Comment:
i would suggest tbh doing something different here; instead of using the same storage class from elsewhere, name the variable to match the intent, which is to set the metastore storage class, and set the var explicitly elsewhere to some value. It might happen that we'd use the same in multiple spots, but in the future, if there's ever a change to a different class which has an odd side effect, we can make changes in one spot, vs having to fiddle and search around in the depths of the roles?

Reply:
@mikehollinger maybe what we can do then is to set `"cpd_block_storage_class: "{{ cpd_storage_class }}"` [here](https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/cp4d_install/defaults/main.yml#L14), making a comment about this and removing block storage usage from our ansible docs for now. That way, if we ever need to change it, it would be in this single place. 
The reason why I think it's not a good idea to set explicitly a value for it is that it would only work for ROKS, but we could use this playbook for other types of envs like Fyre/Quickburn and even AWS/ROSA in the future, so we need to leave it flexible as we can. What u think?